CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- FIX: Do not provide content-type to delete and get http request (since none body is provided)
```

Comment:
Probably this entry is not needed, as it is a fix of the existing "Fix: replace request obsolete library by got"